save-frontend/build.gradle.kts

```diff
@@ -68,13 +68,25 @@ kotlin {
             implementation(npm("@fortawesome/free-solid-svg-icons", "5.15.3"))
             implementation(npm("@fortawesome/free-brands-svg-icons", "5.15.3"))
             implementation(npm("@fortawesome/react-fontawesome", "^0.1.16"))
+            implementation(npm("@mui/material", "^5.9.1"))
```

Comment:
@petertrr does it look similar to https://codesandbox.io/s/react-tsparticles-bg3tsk?file=/package.json